x-pack/test/functional/config.js

```diff
@@ -146,9 +146,6 @@ export default async function({ readConfigFile }) {
       uptime: {
         pathname: '/app/uptime',
       },
-      apm: {
-        pathname: '/app/apm',
-      },
```

Comment:
the duplicate is declared below https://github.com/elastic/kibana/pull/64929/files#diff-6efcd30a7a7fdd7bdab17829239782bcR192